### ISSUE TITLE
 HP:0100632 Pulmonary sequestration vs HP:0010960 Bronchopulmonary sequestration

### ISSUE THREAD

**Issue by kanems** [obophenotype/human-phenotype-ontology#10294]
**HPO term**

 HP:0100632 Pulmonary sequestration vs HP:0010960 Bronchopulmonary sequestration

**Reason for deprecation**
I am not sure if these are the same or not? (One issue from the lot on #9626, splitting out so it can be managed separately)

**Suggested replacement or term to consider**
It would seem one or the other should be used in HPO, unless they are different (in which case I will ask UMLS to split them out into separate CUIs)

**Comment by pnrobinson** [comment]
@mgriese 
is there a difference between Pulmonary and Bronchopulmonary sequestration?

**Comment by mgriese** [comment]
terms are often used synonymous; a sequester is lung tissue not connected to the bronchial tree. Thus the term pulmonary S. appears more approrpiate than bronchopulmonary S.. Pulmonary S is retrieved 3942 times in Pubmed and Bronchopulmonary 2336 and less frequently cited lately.
=> practically no difference between terms; pulmonary S is superior.

**Comment by pnrobinson** [comment]
Thanks!
I am merging the two terms and keeping
HP:0100632 Pulmonary sequestration 
as the primary ID and label.

I notice that each term had two children, which are also synonyms
[Intrapulmonary sequestration HP:0011278](https://hpo.jax.org/app/browse/term/HP:0011278) == [Intralobar sequestration HP:0010961](https://hpo.jax.org/app/browse/term/HP:0010961)
and
[Extrapulmonary sequestrum HP:0006544](https://hpo.jax.org/app/browse/term/HP:0006544) == [Extralobar sequestration HP:0010962](https://hpo.jax.org/app/browse/term/HP:0010962)

with a little research it is clear these are also synonyms. I am merging them and using the labels

Intralobar pulmonary sequestration HP:0011278
Extralobar pulmonary sequestration HP:0006544
